src/iconsManifest/supportedExtensions.ts

```diff
@@ -2558,8 +2558,9 @@ export const extensions: IFileCollection = {
     },
     {
       icon: 'markdownlint',
-      extensions: ['.markdownlint.json'],
-      filename: true,
+      extensions: [],
+      filenamesGlob: ['markdownlint', 'markdownlint-cli2'],
```

Comment:
I would not use filename globs in this case as the extensions needed for the `.markdownlint-cli2.*` are a reduce set.

Anyway, even with this implementation, we're missing `filename: true`  and the initial dot in the filenames.

Reply:
What's the right way you rekon?
just adding them all manually as extensions?